src/pages/requests/RequestSummary.tsx

```diff
@@ -70,13 +74,21 @@ const RequestSummary = observer(({ props }) => {
 
   return (
     <>
-      {/*<table className={"table table-striped"}>*/}
-      {/*  <tbody>*/}
-      {/*{*/}
-      {/*    stringFields*/}
-      {/*}*/}
-      {/*  </tbody>*/}
-      {/*</table>*/}
+      {showDownloadModal && (
+        <DownloadModal
+          loader={() => Promise.resolve(JSON.stringify(data))}
```

Comment:
@shalabhsuman @Jennnyyy note that this data is the entire request object with samples in a sub node, so you'll need to pluck off the sample data when you get around to sending it to CSV